src/cls/IPM/Utils/Update/Controller.cls

```diff
@@ -61,4 +44,39 @@ ClassMethod ApplyUpdatesInOrder(
     quit tSC
 }
 
+ClassMethod ApplySingleModuleUpdatesInOrder(moduleName As %String) As %Status [ Private ]
+{
+}
+
+/// Get list of dependency modules in order of how we want to update them.
+/// We want to perform updates in reverse order of dependencies depth.
+/// i.e. Call update steps for highest depth dependent module all the way up back the base module being updated
+ClassMethod GetProcessModuleList(module As %IPM.Storage.Module) As %List [ Private ]
```

Comment:
Nit: Turn `Process` into `Processing` to avoid confusion with the word process which is an OS-level thing